core/src/main/groovy/noe/tomcat/configure/ConnectorConfiguratorTomcat.groovy

```diff
@@ -56,7 +56,7 @@ class ConnectorConfiguratorTomcat {
       createNewConnector(new ConnectorAttributesTransformer(connector).secureHttpConnector())
     }
 
-    defineRedirectPorts(connector.getPort())
```

Comment:
While creating a new secureHttpConnector, inside server.xml in the end of file was inserting a small block of this code "<Connector redirectPort=8443>" which was making tomcat failed on start up.When commenting it out, this block of code was absent and tomcat had no issues during start up.Its behaviour is kinda random that's why i commented it out.Would like to hear opinions about this

Reply:
Usually, you want redirection from an insecure connection to a secure one, if it's possible(when you enable a secure connector). That's probably the reason why it's there. However, I haven't checked the code so I can't really tell why it's causing an issue or why it's there.

Reply:
I'm seeing your point. This connection between insecure and secure is implemented correctly.Therefore is being added an extra connector with the content i've mentioned. I have made a doc with notes which i can share with you